code/__DEFINES/jobs.dm

```diff
@@ -100,6 +100,54 @@
 #define JOB_DISPLAY_ORDER_AI 34
 #define JOB_DISPLAY_ORDER_CYBORG 35
 
+
+#define JOB_STRING_ASSISTANT    "Assistant"
```

Comment:
if you are adding these defines replace all appropriate uses in the codebase of the strings with the defines. if you are using them in just 1 place its useless and stupid

Reply:
I believe those were explicitly requested by Archanial

Reply:
Fikou is right
also instead of "JOB_STRING" just call it "JOB_"

Reply:
since it touches a lot of codes into the defines, they should be done in another PR, not this PR.
and now defines are "JOB_"